npm/react/src/mount.ts

```diff
@@ -50,7 +50,12 @@ export const mount = (jsx: React.ReactNode, options: MountOptions = {}) => {
   .then(() => {
     const reactDomToUse = options.ReactDom || ReactDOM
 
-    const el = document.getElementById(ROOT_ID)
+    let el = document.getElementById(ROOT_ID)
+
+    if (!el) {
+      el = document.createElement('div')
+      el.id = ROOT_ID
+    }
```

Comment:
I'm pretty concerned about this change. We should fix the problem, rather than simply inserting a new element

Reply:
I wish I could figure out what exactly is going on. Some specific combination of dependencies leads to the root element not existing for reasons I can't figure out :(

Reply:
It's fixed